.buildkite/pipeline.yaml

```diff
@@ -0,0 +1,60 @@
+steps:
+  - label: ":eslint: Lint"
+    command:
+      - "yarn install"
+      - "yarn lintwithexclusions"
+    plugins:
+      - docker#v3.0.1:
+          image: "node:10"
+
+#  - label: ":chrome: End-to-End Tests"
```

Comment:
Maybe we can use a more generic emoji like `:chains:` or `:link:` or... something else. 😉